src/app/components/question-approval-page/question-approval-page.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { QuestionDto } from 'src/app/models/question-dto';
+import { QuestionService } from 'src/app/services/question.service';
+import { FormControl, FormGroup } from '@angular/forms';
+@Component({
+  selector: 'app-question-approval-page',
+  templateUrl: './question-approval-page.component.html',
+  styleUrls: ['./question-approval-page.component.scss']
+})
+export class QuestionApprovalPageComponent implements OnInit {
+  question: QuestionDto =  QuestionDto.createDtoWithNonEmptyFields();
```

Comment:
rather than set this here and then modify it as a side-effect of a function later, it would be better to set it empty first and then within ngOnInit, give it its value. That ensures that any visual components displaying its values can wait for it to be initialised before displaying.